src/main/g8/$if(namespaced.truthy)$src$else$.$endif$/test/$if(namespaced.truthy)$resources$else$.$endif$/$if(namespaced.truthy)$$package$$else$.$endif$/$if(namespaced.truthy)$$name__camel$$else$.$endif$/Test$name__Camel$.tdml

```diff
@@ -18,21 +18,22 @@ limitations under the License.
 -->
 !$
 
-<tdml:testSuite suiteName="$name;format="Camel"$" description="$name;format="Camel"$ tests"
+<testSuite suiteName="$name;format="Camel"$" description="$name;format="Camel"$ tests"
   xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" 
-  xmlns:fn="http://www.w3.org/2005/xpath-functions" 
-  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData" 
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+  xmlns="http://www.ibm.com/xmlns/dfdl/testData"
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   defaultRoundTrip="onePass">
 
```

Comment:
So the tricky part is when you embed either a DFDL schema in the TDML, or a test case has an expected infoset embedded in the XML. In that case the default namespace being the TDML namespace gets in the way. 

One can make it work, by having tdml:defineSchema redefine the default namespace, and for the infoset via clever use of xmlns="" to turn off default namespace. 

Neither of those features add a lot of value to TDML, and as is they're hard to use, so I'm inclined to not steer people toward them. 

I'm planning for a TDML2 which avoids the difficulty there by being defined the way languages should be, which is using elementFormDefault 'unqualified', so that no tdml: prefixes on the child elements are present or allowed. 

For users and TDML v1 (what we have now) I'm thinking we should always put infosets into files.